Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataParametersFactory.cs

```diff
@@ -115,13 +114,52 @@ public IDictionary<string, Parameter> Create(bool isCompositeContext)
                     {
                         name = "totalCount",
                         description =
-                            "Indicates if the total number of items available should be returned in the 'Total-Count' header of the response.  If set to false, 'Total-Count' header will not be provided.",
+                            "Indicates if the total number of items available should be returned in the 'Total-Count' header of the response.  If set to false, 'Total-Count' header will not be provided. Must be false when using key set paging.",
                         @in = "query",
                         type = "boolean",
                         required = false,
                         @default = false
                     }
                 );
+
+                parameters.Add(
+                    "pageToken", new Parameter
+                    {
+                        name = "pageToken",
+                        description = "Applicable when using key set paging. The token of the page to retrieve.",
+                        @in = "query",
+                        type = "string",
+                        required = false
+                    }
+                );
+
+                parameters.Add(
+                    "pageSize", new Parameter
+                    {
+                        name = "pageSize",
+                        description = "Applicable when using key set paging. The maximum number of records to retrieve in the page.",
+                        @in = "query",
+                        type = "integer",
+                        format = "int32",
+                        minimum = 0,
+                        required = false
+                    }
+                );
+
+                parameters.Add(
+                    "numberOfPartitions", new Parameter
+                    {
+                        name = "number",
+                        description = "The number of evenly distributed partitions to provide across the matching results.",
```

Comment:
This text appears here:

![image](https://github.com/user-attachments/assets/c688f14d-0357-4ab1-a876-90f6149e91c8)

Reply:
This change looks good

Reply:
Changed.